test/src/kbasesearchengine/test/main/SearchMethodsTest.java

```diff
@@ -277,5 +293,138 @@ private void searchTypesCheckMatchFilter(
         // if we want to check search time, need to mock a Clock. Don't bother for now.
         // assertThat("incorrect objects", res.getSearchTime(), is(20));
     }
-    
+
+
+
+
+    @Test
+    public void searchObjectResult() throws Exception {
```

Comment:
For future reference this should really be split up into separate tests. It's crazy long and the various cases could be a test each.

Reply:
its long mainly b/c of set up. how would you suggest splitting this up?

Reply:
Talked about this in person